activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -999,7 +999,7 @@ def insert_versions_sql(versions) # :nodoc:
 
         if supports_multi_insert?
           sql = "INSERT INTO #{sm_table} (version) VALUES "
```

Comment:
Let's add a newline here too, so we don't leave the first version dangling on the end of the line.